pentasharp/Models/Entities/Taxi.cs

```diff
@@ -36,6 +36,7 @@ public class Taxi
         public ICollection<TaxiReservations> TaxiReservations { get; set; } = new List<TaxiReservations>();
 
         public ICollection<TaxiBookings> TaxiBookings { get; set; } = new List<TaxiBookings>();
+        public bool IsDeleted { get; set; }
```

Comment:
leave one space in between properties and remove data annotations since we agreed on not using them like this anymore

ctrl + a , ctrl + k + f should format the code